build/illume.js

```diff
@@ -11,17 +11,15 @@ var _q = require("@standard-library/q");
 var _kefir = require("kefir");
 
 var offsetAbove = function offsetAbove(y, element) {
-  return y > $(element).offset().top;
+  return y > element.offsetTop;
```

Comment:
Hello, I am back! 

`getBoundingClientRect` is something I've seen mentioned a couple of times:
https://developer.mozilla.org/en-US/docs/Web/API/Element/getBoundingClientRect

There are some caveats. There are a few arguments on this page:
http://stackoverflow.com/questions/18953144/how-do-i-get-the-offset-top-value-of-an-element-without-using-jquery

Not finding a lot of libs for this. Here's one:
https://www.npmjs.com/package/document-offset